examples/reference-erc20/ponder-env.d.ts

```diff
@@ -1,27 +1,15 @@
+/// <reference types="ponder/virtual" />
+
+declare module "ponder:register" {
```

Comment:
What's the full list of names you considered here? Want to get this right, because it's going to show up in a LOT of code snippets for a long time :)

Edit: Oh, I figured this was the name that would replace "@/generated" but now I see that's different. Do you think we should rename "@/generated" in this release? I lean yes.

Reply:
Yes I think we should replace `@/generated`, but I haven't done it yet. 

This `ponder:register` is an entirely internal thing that is used to pass type information back into the `packages/core/src/types.d.ts`.